.gitlab-ci.yml

```diff
@@ -200,8 +200,9 @@ test:unit:mac:
     - mac-runner
 
 # Test that we can build with the golang version of the oldest supported yocto LTS release
```

Comment:
I suppose this line is invalid now as Kirkstone is still supported, right?
```diff
+ # Test that we can build with the golang version of the oldest supported yocto LTS release
- # Test that we can build with the golang version of the newest supported yocto LTS release
```

Reply:
I wanted to keep it, as to mark that supporting the oldest is the policy and we make an exception in this repo at this time due to the mentioned ticket (next line).

If you find it confusing I would remove the whole line - just modifying from "oldest" to "newest" I don't want to do because that can lead to confusion in one year from now.